### ISSUE TITLE
Export images with higher resolution

### ISSUE THREAD

**Issue by adrianusler** [Molara-Lab/Molara#326]
In think that this requires something like a virtual openGLWidget that is larger than the displayed one. One could then maybe call `makeCurrent()` on this one and grab the framebuffer from there? I'll see what can be done.

**Update**: I found a solution and will implement this soon!
**Update**: See PR #373. But I need to implement some tests before we can merge it.

**Comment by Michel-Heinz** [comment]
It might be possible to render to a separate framebuffer. However I do not know if there are any routines to export this framebuffer as png.

**Comment by adrianusler** [comment]
@Michel-Heinz apparently, the openGL-context must be part of a window. So I guess I have to add a "virtual" (i.e., permanently hidden) Gui form to the mix and render images on that when it's needed.

**Comment by adrianusler** [comment]
Technically, this is a duplicate of #221. But since we have had a discussion here, I will close #221 (even though it is older).
